framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -1081,7 +1086,47 @@ protected void printOrStoreMessage(
             String message,
             Tree source,
             CompilationUnitTree root) {
+        StackTraceElement[] trace = Thread.currentThread().getStackTrace();
+        printOrStoreMessage(kind, message, source, root, trace);
+    }
+
+    /**
+     * printOrStoreMessage method with an added stack trace argument. The stack trace is printed
```

Comment:
Like for other overloaded methods, start out with the normal description of what the method is for and then describe the differences.
Also, `@see` cross-references can be useful.

Reply:
I have updated the documentation in my latest commit.